NEWS.md

```diff
@@ -1,3 +1,7 @@
+# Changes in version 2024.10.10 (PR#154)
+
+  - Updated `animist.js` to use classes for source links, resolving issue #154.
```

Comment:
now the issue number is incorrect please revise

Reply:
https://github.com/animint/animint2/pull/154/commits/703d0656ad6b89d0f8ee52aaa175b2df7be8134c

this commit was correct, but I have updated to my last commit. Sorry for the misunderstanding.